res/skins/LateNight/effect_unit.xml

```diff
@@ -93,11 +93,12 @@
                   <Text>ON</Text>
                 </State>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
```

Comment:
I'm boggling a bit as to why there were two "enabled" connections here. @ywwg do you remember?